hack/make-release-artifacts.sh

```diff
@@ -89,6 +89,20 @@ mk_istio_manifests() {
     echo '# [END servicemesh_release_istio_manifests_microservices_demo]'
 }
 
+mk_kustomize_base() {
+  for file_to_copy in ./kubernetes-manifests/*.yaml
+  do
+    cp ${file_to_copy} ./kustomize/base/
+
+    service_name="$(basename "${file_to_copy}" .yaml)"
+    image="$REPO_PREFIX/$service_name:$TAG"
+
+    pattern="^(\s*)image:\s.*${service_name}(.*)(\s*)"
+    replace="\1image: ${image}\3"
+    gsed --in-place --regexp-extended "s|${pattern}|${replace}|g" ./kustomize/base/${service_name}.yaml
+  done
+}
```

Comment:
The `basename "${file_to_copy}" .yaml` command is used to turn something similar to:
`./kubernetes-manifests/adservice.yaml`
into
`adservice`